doc/classes/AABB.xml

```diff
@@ -26,6 +26,7 @@
 			<param index="0" name="from" type="AABB" />
 			<description>
 				Constructs an [AABB] as a copy of the given [AABB].
+				[b]Note:[/b] Not available in C#.
```

Comment:
Perhaps. This is an "early stop gap" of sorts so that you stop reading the note as soon as it doesn't interest you.
```suggestion
				[b]Note:[/b] In C#, this constructor is not available.
```